jenkins/job-configs/bootstrap-maintenance.yaml

```diff
@@ -80,6 +80,10 @@
             </hudson.plugins.parameterizedtrigger.TriggerBuilder>
     triggers:
     - timed: '{frequency}'
+    wrappers:
+    - timeout:
+        timeout: 30
```

Comment:
I think 30m for the "all" jobs is not enough, since the maintenance jobs won't get scheduled on a node until after a previous job finishes. maybe 30m for the maintenance job, 3h for the "all" jobs?